I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/test_case/test_case_setup.rpy", line 483, in script
    call screen maddies_house_scene
  File "renpy/common/000statements.rpy", line 670, in execute_call_screen
    store._return = renpy.call_screen(name, *args, **kwargs)
  File "game/test_case/test_case_setup.rpy", line 26, in execute
    screen inventoryItemMenu(item):
  File "game/test_case/test_case_setup.rpy", line 26, in execute
    screen inventoryItemMenu(item):
  File "game/test_case/test_case_setup.rpy", line 28, in execute
    frame:
  File "game/test_case/test_case_setup.rpy", line 46, in execute
    python:
  File "game/test_case/test_case_setup.rpy", line 47, in <module>
    print(inv_item)
NameError: name 'inv_item' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/test_case/test_case_setup.rpy", line 483, in script
    call screen maddies_house_scene
  File "C:\Users\Quinn\Documents\renpy-8.1.1-sdk\renpy\ast.py", line 2259, in execute
    self.call("execute")
  File "C:\Users\Quinn\Documents\renpy-8.1.1-sdk\renpy\ast.py", line 2241, in call
    return renpy.statements.call(method, parsed, *args, **kwargs)
  File "C:\Users\Quinn\Documents\renpy-8.1.1-sdk\renpy\statements.py", line 342, in call
    return method(parsed, *args, **kwargs)
  File "renpy/common/000statements.rpy", line 670, in execute_call_screen
    store._return = renpy.call_screen(name, *args, **kwargs)
  File "C:\Users\Quinn\Documents\renpy-8.1.1-sdk\renpy\exports.py", line 3345, in call_screen
    rv = renpy.ui.interact(mouse="screen", type="screen", roll_forward=roll_forward)
  File "C:\Users\Quinn\Documents\renpy-8.1.1-sdk\renpy\ui.py", line 299, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "C:\Users\Quinn\Documents\renpy-8.1.1-sdk\renpy\display\core.py", line 3579, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "C:\Users\Quinn\Documents\renpy-8.1.1-sdk\renpy\display\core.py", line 4052, in interact_core
    root_widget.visit_all(lambda d : d.per_interact())
  File "C:\Users\Quinn\Documents\renpy-8.1.1-sdk\renpy\display\core.py", line 681, in visit_all
    d.visit_all(callback, seen)
  File "C:\Users\Quinn\Documents\renpy-8.1.1-sdk\renpy\display\core.py", line 681, in visit_all
    d.visit_all(callback, seen)
  File "C:\Users\Quinn\Documents\renpy-8.1.1-sdk\renpy\display\core.py", line 681, in visit_all
    d.visit_all(callback, seen)
  File "C:\Users\Quinn\Documents\renpy-8.1.1-sdk\renpy\display\screen.py", line 476, in visit_all
    callback(self)
  File "C:\Users\Quinn\Documents\renpy-8.1.1-sdk\renpy\display\core.py", line 4052, in <lambda>
    root_widget.visit_all(lambda d : d.per_interact())
  File "C:\Users\Quinn\Documents\renpy-8.1.1-sdk\renpy\display\screen.py", line 487, in per_interact
    self.update()
  File "C:\Users\Quinn\Documents\renpy-8.1.1-sdk\renpy\display\screen.py", line 680, in update
    self.screen.function(**self.scope)
  File "game/test_case/test_case_setup.rpy", line 26, in execute
    screen inventoryItemMenu(item):
  File "game/test_case/test_case_setup.rpy", line 26, in execute
    screen inventoryItemMenu(item):
  File "game/test_case/test_case_setup.rpy", line 28, in execute
    frame:
  File "game/test_case/test_case_setup.rpy", line 46, in execute
    python:
  File "game/test_case/test_case_setup.rpy", line 47, in <module>
    print(inv_item)
NameError: name 'inv_item' is not defined

Windows-10-10.0.19045 AMD64
Ren'Py 8.1.1.23060707
detective test 1.2
Sun Jan 14 00:48:24 2024
